cypress/e2e/44_rolesManageBounties.cy.ts

```diff
@@ -0,0 +1,83 @@
+describe('carol manage organization bounties', () => {
+  it('should carol manage organization bounties', () => {
+    const OrgName = 'Org_Name';
+
+    const org: Cypress.Organization = {
+      loggedInAs: 'alice',
+      name: OrgName,
+      description: 'An organization focused on amazing projects.',
+      website: 'https://amazing.org',
+      github: 'https://github.com/amazing'
+    };
+
+    const bounty: Cypress.Bounty = {
+      organization: OrgName,
+      title: 'Role Manage Bounty',
+      category: 'Web development',
+      coding_language: ['Typescript', 'Javascript', 'Lightning'],
+      description: 'This is available',
+      amount: '123',
+      tribe: 'Amazing Org Tribe',
+      estimate_session_length: 'Less than 3 hour',
+      estimate_completion_date: '09/09/2024',
+      deliverables: 'We are good to go man',
+      assign: ''
+    };
+
+    cy.login(org.loggedInAs);
+    cy.wait(1000);
+
+    cy.create_org(org);
+    cy.wait(1000);
+
+    cy.contains(org.name).contains('Manage').click();
+    cy.wait(1000);
+
+    cy.contains('Add User').click();
+    cy.wait(1000);
+
+    cy.get('input').type('carol');
+    cy.wait(1000);
+
+    cy.contains('Select').click();
+    cy.wait(1000);
+
+    cy.get('#sphinx-top-level-overlay').should('be.visible');
+
+    cy.get('#sphinx-top-level-overlay').within(() => {
+      cy.contains('Add User').click();
+      cy.wait(1000);
+    });
+    cy.wait(1000);
+
+    cy.get('#sphinx-top-level-overlay').within(() => {
+      cy.contains('Assign').click();
+      cy.wait(1000);
+    });
+    cy.wait(1000);
+
+    cy.logout(org.loggedInAs);
+    cy.wait(1000);
+
+    cy.login('carol');
+    cy.wait(1000);
+
+    cy.contains('carol').click({ force: true });
+    cy.wait(1000);
+
+    cy.contains(OrgName).contains('Manage').should('exist');
+    cy.wait(1000);
+
+    cy.create_bounty(bounty);
+    cy.wait(1000);
+
+    cy.contains(bounty.title).click();
+    cy.wait(1000);
+
+    cy.contains('Mark as Paid').should('exist');
```

Comment:
@AbdulWahab3181 what we are asserting for is `Pay Bounty` not `Mark as Paid`

to achieve this change this on CodingBounty.tsx

```sh
 const isOwner =
    { ...person }?.owner_alias &&
    ui.meInfo?.owner_alias &&
    { ...person }?.owner_alias === ui.meInfo?.owner_alias;

  const hasAccess = isOwner || userBountyRole;
  const payBountyDisable = !isOwner && !userBountyRole;

  const showPayBounty =
    !bountyPaid && !invoiceStatus && hasAccess;
```

update the showPayBounty to this.

Reply:
@AbdulWahab3181 also remove these comments, this change will make it redundant

   {/**
       * LNURL AUTH users alias are their public keys
       * which make them so longer
       * A non LNAUTh user alias is shorter
  */}